lightly_studio/tests/core/test_add_samples.py

```diff
@@ -341,6 +325,35 @@ def test_tag_samples_by_directory_tag_depth_1(
     assert sample_filename_to_tags["root_img.png"] == set()
 
 
+def test_add_samples_from_path_calls_tag_samples_by_directory(
```

Comment:
Major: This belongs into `lightly_studio/tests/core/test_dataset__path.py`, as it is testing a function in `dataset.py`, not in `add_samples.py`.
As `test_dataset.py` would become to large if we put all tests into it, we have split it into different test files, including `test_dataset__path.py` for testing `Dataset.add_samples_from_path`.

You can move it easily in VSCode via right-clicking on the function name -> Refactor -> Move symbol to. This will also move all imports.